lib/ecto/schema.ex

```diff
@@ -650,6 +650,26 @@ defmodule Ecto.Schema do
         end
       end
 
+  A query with *only* where clauses can be provided, instead of a related schema, and the where clauses
```

Comment:
I think the current description focuses too much on the implementation. My suggestion:

> A query can also be given instead of a schema. Querying, joining or preloading the association will use the given query. Currently only `where` clauses can be provided in queries. Let's see an example:
> ...
> Note building the association does not consider the query filters. For example, if the given query requires the `active` field of the associated records to be true, building such association won't automatically set the `active` field to true.

Reply:
Sounds great :) I'll make those changes this evening